modules/gatsby/src/components/layout/table-of-contents.jsx

```diff
@@ -55,7 +56,7 @@ const renderRoute = (route, i, pathname, depth) => {
   if (route.chapters) {
     const name = route.title;
     return (
-      <div key={i} style={{marginLeft: 10 * depth}}>
+      <div key={i} style={{ marginLeft: 10 * depth }}>
```

Comment:
Yep, we definitely want to get prettier in place so we don't need to bike shed over spacing...